### DISCUSSION TITLE
View manual mesh levelling result report

### DISCUSSION THREAD

**Discussion by melvinmicallef** [Jyers/Marlin#56] [Q&A] [answered]
Using the Ender 3 v2 'Manual Mesh 3x3' config, is it possible to query a mesh report through gcode to be used in octoprint plugin https://plugins.octoprint.org/plugins/bedlevelvisualizer/ ? 

Could not find it in https://marlinfw.org/docs/gcode/G029-abl-bilinear.html

**Comment by Jyers** [comment, marked as answer]
You couldn't find it because its not part of the G29 command. Its tied into M420. I believe its M420 V1 to be specific

**Comment by wbvczar** [comment]
I used the manual mesh levelling also (5x5).  When I went into the Bed Visualizer and had it get the mesh info, it seemed like it was not doing anything, but I left it for almost a minute, then the print head moved to the first position and then I had the visualizer data.
